python/lib/communication/dmod/communication/maas_request/distribution.py

```diff
@@ -1,8 +1,28 @@
-class Distribution:
+from dmod.core.serializable import Serializable
+
+from typing import Literal
+
+
+class DistributionBounds(Serializable):
+    minimum: int = 0
+    maximum: int = 0
+    distribution_type: Literal["normal"] = "normal"
+
+    class Config:
+        feilds = {
+            "distribution_type": {"alias": "type"},
+            "minimum": {"alias": "min"},
+            "maximum": {"alias": "max"},
+        }
+
+
+class Distribution(Serializable):
```

Comment:
This sounds a lot like your embedding / merging models question from earlier. I refactored it in this way to conserve the shape of the serialized `Distribution` class (i.e. `{"distribution": {"type": "normal", "min": 0, "max": 0}}`). Unfortunately to my knowledge `pydantic` does have a way to create a tagged model. I am happy to embed the `DistributionBounds` model within the `Distribution` class like I showed in the above example if you'd like me to?

Reply:
It was the same idea as before where instead of `{"distribution": {"type": "normal", "min": 0, "max": 0}}`, you'd just have `{"type": "normal", "min": 0, "max": 0}`. I think that sort of change is out of scope of this ticket, so the question is: is this something we might want to refactor _later_ and should we add a ticket?

Reply:
Yeah, @christophertubbs this is something we would want to handle in another effort. Please add a ticket for this, thanks!

Reply:
Just noticed that `fields` is misspelled. That **definitely** needs to be fixed.

Reply:
The structure of the `NWMConfig` class may need to be further investigated and it may go beyond distilling `DistributionBounds` into `Distribution`. By reading the source, it looks like `NWMConfig.parameters` is a mapping between string names and `Scalar`s and `Distribution`s, but it is marked as a list of strings that is used to mark allowable keys for the `Scalar`s and `Distribution`s. The class variable there should probably be renamed to something like `NWMConfig.allowable_parameters` so the name doesn't clash between classes and instances.

Reply:
Opened ticket #340